connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -388,14 +398,13 @@ export async function syncZendeskTicketsActivity({
     dataSourceId: dataSourceConfig.dataSourceId,
   };
 
-  const { accessToken, subdomain } = await getZendeskSubdomainAndAccessToken(
-    connector.connectionId
+  const zendeskApiClient = createZendeskClient(
+    await getZendeskSubdomainAndAccessToken(connector.connectionId)
   );
-  const zendeskApiClient = createZendeskClient({
-    token: accessToken,
-    subdomain,
+  await changeZendeskClientSubdomain(zendeskApiClient, {
```

Comment:
Do we really need that change function? Can't we pass the brandId do getZendeskSubdomainAndAccessToken ?

Reply:
There are some cases where we initialize the `ZendeskClient` once and change its subdomain repeatedly so we need 2 functions. `getZendeskSubdomainAndAccessToken` currently only talks to OAuth and I don't want a function that optionally also fetches Zendesk to get the brand subdomain

Reply:
👍